GTG/gtk/browser/main_window.py

```diff
@@ -400,6 +400,12 @@ def _init_signal_connections(self):
         self.connect('notify::default-width', self.on_window_resize)
         self.connect('notify::default-height', self.on_window_resize)
 
+        for p in PANE_STACK_NAMES_MAP.keys():
+            pane = self.stack_switcher.get_stack().get_child_by_name(p).get_first_child()
```

Comment:
nit: Maybe at some point we can have a method that does this like `get_pane_by_name()`

Reply:
Good idea!